bootstrap-demo/kubernetes-api-java/src/main/java/akka/cluster/bootstrap/demo/DemoApp.java

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Could we just put this in the same project to avoid the non code related duplication?